website/configs/resources.json

```
@@ -1,5 +1,13 @@
 {
   "data": [
+    {
+      "heading": "How To Build an Advanced Multipart Component with Chakra UI",
+      "type": "blog",
+      "tags": ["Vite", "Advanced", "Theming" "v1.6"],
```

you're missing a `,` here

```suggestion
      "tags": ["Vite", "Advanced", "Theming", "v1.6"],
```